R/paths.R

```diff
@@ -17,6 +17,9 @@ renv_paths_common <- function(name, prefixes = NULL, ...) {
     Sys.getenv(envvar, unset = NA) %NA%
     renv_paths_root(name)
 
+  if (identical(name, "cache") && grepl(";", root, fixed = TRUE))
```

Comment:
Could we add a brief comment here outlining the motivation? Also, I wonder if this should be `:` rather than `;`, or `[:;]`, or `.Platform$path.sep`.

Reply:
I added a comment, let me know if it needs to be more or less detailed. Also, I opted for `[:;]` so that code that creates the environment variables can be written in a platform-agnostic manner.